.github/workflows/build-android-examples.yaml

```
@@ -0,0 +1,18 @@
+jobs:
+  build-wallet-app-example:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Flutter setup
+        uses: ./.github/actions/flutter-setup
+      - name: Java/Android setup
+        uses: ./.github/actions/android-setup
+      - uses: bluefireteam/melos-action@v3
+      - name: Build wallet_app example
+        run : flutter build appbundle
```

yamllint (1.35.1)

[warning] too many spaces before colon

(colons)